packages/components/src/components/Table/Table.tsx

```diff
@@ -36,15 +36,25 @@ const Table = React.forwardRef<HTMLTableElement, TableProps>(
 
     return (
       <TableContext.Provider value={tableContext}>
-        <StyledTable
-          borderCollapse="separate"
-          ref={ref}
-          tableLayout={tableLayout}
-          w="100%"
-          {...props}
+        <Box.div
+          backgroundColor="colorBackground"
+          borderColor="colorBorderWeaker"
+          borderRadius="borderRadius20"
+          borderStyle="borderStyleSolid"
+          borderWidth="borderWidth10"
+          h="100%"
+          overflow="scroll"
         >
-          {children}
-        </StyledTable>
+          <StyledTable
+            borderCollapse="separate"
+            ref={ref}
+            tableLayout={tableLayout}
+            w="100%"
```

Comment:
what if we add a backgroundColor property to the component, adding `bgPrimary` (new token who dis 🎉) as default? So it would still be possible to change it and the change would still be made?

Reply:
Changed! Thank you :)